libs/components/lookup/src/lib/modules/lookup/lookup-show-more-modal.component.ts

```diff
@@ -80,6 +82,7 @@ export class SkyLookupShowMoreModalComponent
 
   constructor(
     public modalInstance: SkyModalInstance,
+    @Inject(SKY_SHOW_MORE_NATIVE_PICKER_CONTEXT)
```

Comment:
I think it makes sense to keep `SkyLookupShowMoreNativePickerContext` as a class since it also serves as an injection token.